package.json

```diff
@@ -1,13 +1,26 @@
 {
-  "name": "nuxt-device-detect",
-  "version": "1.1.5",
+  "name": "@nuxtjs/device",
+  "version": "1.2.0",
   "description": "",
-  "main": "index.js",
+  "main": "lib/module.js",
   "types": "types.d.ts",
-  "repository": "https://github.com/dotneet/nuxt-device-detect",
+  "repository": "https://github.com/nuxt-community/device-module",
```

Comment:
This can be simplified to `nuxt-community/device-module`.

PS would you please run `npx sort-package-json`?

Reply:
Thanks! I'll do that.